app/assets/stylesheets/forever_style_guide/modules/_button.scss

```diff
@@ -57,7 +58,7 @@
   }
 
   // creates a button style for each color in core color dictionary
```

Comment:
minor note but this should be updated to color dictionary rather than core color dictionary